jmix-pivottable/pivottable-flowui/src/main/java/io/jmix/pivottableflowui/export/PivotTableExporterImpl.java

```diff
@@ -36,22 +40,30 @@
  */
 @Component("pvttbl_PivotTableExporter")
 @Scope(BeanDefinition.SCOPE_PROTOTYPE)
-public class PivotTableExporterImpl implements PivotTableExporter {
+public class PivotTableExporterImpl implements PivotTableExporter, ApplicationContextAware, InitializingBean {
 
     public static final Set<Renderer> supportedRenderers = Collections.unmodifiableSet(Sets.newHashSet(
             Renderer.TABLE, Renderer.TABLE_BAR_CHART, Renderer.HEATMAP, Renderer.ROW_HEATMAP, Renderer.COL_HEATMAP));
 
+    protected ApplicationContext applicationContext;
     protected PivotTableExcelExporter excelExporter;
     protected PivotTable<?> pivotTable;
     protected String fileName;
 
-    public PivotTableExporterImpl(PivotTable<?> pivotTable, PivotTableExcelExporter exporter) {
```

Comment:
We need to keep this constructor and create a new one.

Since this API was not marked as `experimental`, and was not marked as `@Internal`, we do not have the right to change it in a bugfix release (even if no one extended it).

Unfortunately, we will have to keep it and mark it as deprecated, with a comment that we need to use a new constructor.